nais_example.yaml

```diff
@@ -17,6 +17,7 @@ healthcheck: #Optional
     initialDelay: 20
 #Optional. Defaults to NONE.
 #See https://kubernetes.io/docs/concepts/containers/container-lifecycle-hooks/
+leaderElection: false # if true, naisd will sidecar an elector that can be reach on $ELECTOR_PATH
```

Comment:
reachED

Reply:
I also think the sentence should be more about usage than implementation:
If true, a http endpoint will be availableat $ELECTOR_PATH  that will return the current leader. Compare this value with $HOSTNAME to see if the current instance is leader